Library/Homebrew/cmd/desc.rb

```diff
@@ -1,6 +1,12 @@
 #:  * `desc` <formula>:
 #:    Display <formula>'s name and one-line description.
 #:
+#:  * `desc` -a:
```

Comment:
If this remains part of desc it should be `--all`. I think it may make more sense as `brew search --desc` without any other arguments, though.